android/KMEA/app/src/test/java/com/tavultesoft/kmea/packages/PackageProcessorTest.java

```diff
@@ -44,10 +49,10 @@
   private static void createAlternateKMP() throws Exception {
     FileUtils.copyFile(TEST_GFF_KMP_FILE, TEST_GFF_KMP_FILE_ALT);
     try {
-      tempPkgAlt = PackageProcessor.unzipKMP(TEST_GFF_KMP_FILE_ALT);
+      File tempPkgAlt = PackageProcessor.unzipKMP(TEST_GFF_KMP_FILE_ALT);
 
       JSONObject json = PackageProcessor.loadPackageInfo(tempPkgAlt);
-      json.getJSONObject("system").put("fileVersion", "8.0"); // Make it look newer!
+      json.getJSONObject("info").getJSONObject("version").put("description", "1.5"); // Make it look newer!
```

Comment:
I think I'd like to have a separate test file rather than fudging our test like this :) I know it's the same outcome but we can fairly easily construct two versions of the same .kmp.

Reply:
I've separated the test kmp files to:
test_resources/v14/gff_amh_7_test_json.kmp (package v1.4)
test_resources/v15/gff_amh_7_test_json.kmp (package v1.5)

Can @jahorton update `PackageProcessorTest.java`? tbh, I'm not quite following when temporary file names get used.

Reply:
From the standup meeting, it sounds like we want to replace those test kmp files with "slimmer" files (don't need fonts, just metadata and js files)